### PR TITLE
Add link to Nordix Open Labs on the website

### PR DESCRIPTION
**Pull request by fdegir**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -23,10 +23,14 @@ disableKinds = [ "RSS", "sitemap", "taxonomy", "taxonomyTerm" ]
     name = "Wiki"
     url = "https://wiki.nordix.org/"
     weight = 5
+[[menu.main]]
+    name = "Nordix Open Labs"
```
Comment (andrew-est) [inline config.toml:27]: I think this should be nolabs as the "Nordix Open Labs" will be too long on page

**Review by andrew-est — CHANGES_REQUESTED** [review]
I

File: config.toml (hunk 1 above)
Comment (fdegir) [reply config.toml:27]: Yeah, I wasn't sure what to put there.

nolabs might not make sense to people and Nordix Open Labs is too long so I opted for the long name to be explicit.

Comment (andrew-est) [reply config.toml:27]: Thats OK with me

**Review by andrew-est — APPROVED** [review]

**Pull request merged**